.github/workflows/realease-on-pr-merge.yaml

```
@@ -39,17 +39,25 @@ jobs:
         run: |
           git config --global user.name "SeungBin Kim"
           git config --global user.email "luxorienskim@gmail.com"
-      - name: Bump version
+      - name: Bump version and push tag
+        id: bump_version
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:11:16: Double quote to prevent globbing and word splitting

(shellcheck)